src/buffer.rs

```diff
@@ -362,29 +365,14 @@ where Container: ArrayLike<T>, T: Primitive + 'static, PixelType: Pixel<T> + 'st
     /// # Panics
     ///
     /// Panics if `(x, y)` is out of the bounds (width, height)`.
-    pub fn put_pixel(&mut self, x: u32, y: u32, pixel: PixelType) {
+    pub fn put_pixel(&mut self, x: u32, y: u32, pixel: P) {
         *self.get_pixel_mut(x, y) = pixel
     }
-
-    /// Casts the buffer into a dynamically typed image buffer
-    #[experimental]
-    pub fn into_dynamic(self) -> ImageBuffer<Container, T, &'static Any> {
-        ImageBuffer {
-            data: self.data,
-            width: self.width,
-            height: self.height,
-            type_marker: self.type_marker
-
-        }
-    }
```

Comment:
This was useless anyway since I did not implement the accompanying code yet. But I think it should work when implementing Pixel for Any…